addons/mail/static/src/core/common/composer.js

```diff
@@ -309,7 +309,7 @@ export class Composer extends Component {
     }
 
     get sendKeybinds() {
```

Comment:
Not in diff but ideally markup shouldn't be around translated value in OR_PRESS_SEND_KEYBIND but only around the send_keybind itself.

If translation doesn't support markup yet in this version, translated content should be escaped before doing a manual sprintf.